### DISCUSSION TITLE
Minecraft Editor didn't remember last UI size setting

### DISCUSSION THREAD

**Discussion by fraysdev** [Mojang/minecraft-editor#76] [Bugs]
Since UI size 100% is too big for me, I change into 75%, and it's much better 

But when I reopen my Minecraft, it's back to 100%, even though it's say 75%. It's get annoyed since I need to change it every time

**Comment by minecraftchloe** [comment]
Hmm :( I'm not able to reproduce this issue. Tried a few different things - going between Editor, release Minecraft, and non-Editor Preview to see if there was some reset happening.
